docs/Product-Overview.md

```diff
@@ -0,0 +1,72 @@
+commits during the life of the project.
+
+## What exactly is OTP?
+
+The most commonly used OpenTripPlanner component is a server-side Java application that is primarily 
+designed to import map, transit service, and other mobility data sets, input user origins and 
+destinations, and output API endpoints that allow other applications to consume generated trip plans, 
+vector map tiles, stop departures, and other rider information.
+
+In other words, OTP is a backend application that can work with other frontend user interfaces. OTP 
+works with your website, mobile app, physical signage or other applications in order to provide 
+relevant customer information and a meaningful trip plan to riders.
+
+There's no official OTP user interface, although the OpenTripPlanner ecosystem includes several user 
+interface projects. How your OTP looks is entirely up to you.
+
+## What OTP can do for you
+
+### Transit agencies
+
+Transit agencies can use OTP as their public website/app trip planner or as an internal trip planner 
+for their customer service team.
+
+- Fully customizable for use in any frontend
+- High-performance trip planning
+- Fully multimodal routing, including walking, biking, bikeshare, demand-responsive services in 
+combined trip plans rather than presented in alternative trip plans
+- Support for real-time alerts, trip re-routing based on real-time delays
+- Pareto optimal multi-criteria cost calculation allows for dynamic user inputs and requests including 
+mode, time, transfers, and other preferences while delivering balanced results
+- One OTP backend can power both a website and app for use by riders, along with a separate frontend 
+with a different configuration for use by a customer service team
+- Active development by dozens of agencies means new features every year ['(see the OTP common roadmap here)'] (https://docs.google.com/document/d/1mahjShc47UCSY9izBvDAw6eXN6Vy6JdRo42iq7YvwVQ/edit)
+
+### Regions, provinces, and nations
+
+Regional, provincial, or national governments, or any private entity, can set up a trip planner including multiple transit agencies.
+
+- Highly scalable software for large regions
+- Interagency fare calculation
+- Integrates multiple modes including bikeshare and demand-response
+- Can be provided as a service to agencies for use in their websites and apps
+
+### Researchers
+
+While historically, OpenTripPlanner has often been used by researchers for the programmatic calculation 
+of large numbers of trip plans, this is no longer an intended use case of OTP. You are welcome to use 
+OTP for any purpose, but you may also find applications like ['r5'] (https://github.com/conveyal/r5) to 
+be more appropriate to their purposes.
+
+## Talk to an expert about OTP
+
+Everyone interested in OTP is welcome to post questions on the ['OpenTripPlanner-users group'] (https://groups.google.com/g/opentripplanner-users). 
```

Comment:
Could you mention the gitter channel at https://gitter.im/opentripplanner/OpenTripPlanner ?

Reply:
Yes! Kept it brief. Let me know if you think we should specify which is for what type of comms.